### DISCUSSION TITLE
is previewData is used in any files ?

### DISCUSSION THREAD

**Discussion by sagar9990** [chartbrew/chartbrew#244] [Q&A] [answered]
In the Chartbrew client-side code, I've identified a getPreviewData method in both client/src/slice/chart.js and client/src/action/chart.js. However, after searching through the project, I couldn't find any instances where these methods are being used. Should I consider removing this code?

**Comment by razvanilin** [comment]
Hey @sagar9990, the `actions` folder is legacy code from the old redux codebase. Some actions are still not fully migrated, hence the files still being there. Everything in the `slice` folder is the new `redux-toolkit` code. Rather than removing function by function, I think a better approach will be to remove the `action` files entirely when none of the functions there are being used anymore

**Reply by sagar9990** [reply]
Thank you @razvanilin ,

I noticed that in the slice folder chart.js file, there is a getPreviewData method. After searching through the client-side code, it seems this method is not being used anywhere. Should I go ahead and remove this function as well?

**Reply by razvanilin** [reply, marked as answer]
sure, feel free to remove if it's not used
